### PR TITLE
media-plugins/vdr-ddci2: new package

### PR DESCRIPTION
**Pull request by herrnst**
Digital Devices CI Bridge/Flex module support plugin for VDR.

Tested/checked with Repoman 2.3.1 (portage-2.3.3)

### CODE REVIEW

**Comment by herrnst** [comment]
Bugzilla account has since been registered.

File: media-plugins/vdr-ddci2/vdr-ddci2-1.0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	KEYWORDS=""
+	S="${WORKDIR}/${P}"
+else
+	SRC_URI="https://github.com/jasmin-j/vdr-plugin-${VDRPLUGIN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/vdr-plugin-${VDRPLUGIN}-${PV}"
+fi
+
+DESCRIPTION="VDR plugin: DDCI2 - Digital Devices CI support"
+HOMEPAGE="https://github.com/jasmin-j/vdr-plugin-${VDRPLUGIN}"
```
Comment (mgorny) [inline media-plugins/vdr-ddci2/vdr-ddci2-1.0.4.ebuild]: Please don't use variable in HOMEPAGE. People are often clicking that when reading ebuilds.

**Comment by mgorny** [comment]
@gentoo/vdr

File: media-plugins/vdr-ddci2/vdr-ddci2-1.0.4.ebuild (hunk 1 above)
Comment (herrnst) [inline media-plugins/vdr-ddci2/vdr-ddci2-1.0.4.ebuild]: Makes sense. Should I fix this prior to merge (commit-update, squash) or will you do this? (first-time portage contribution, not exactly sure how you handle things like this)

Comment (mgorny) [inline media-plugins/vdr-ddci2/vdr-ddci2-1.0.4.ebuild]: The *fix on merge* label indicates that the developer can fix it while merging. Of course, you can force-push your fix over this if you like; but it's simple enough not to warrant extra work on you while it's easier to fix while preparing the commit for merge.

Comment (herrnst) [inline media-plugins/vdr-ddci2/vdr-ddci2-1.0.4.ebuild]: Thanks for the explanation! Went the forcepush-route and also fixed the copyright year which repoman didn't complain about before...

**Comment by herrnst** [comment]
Hmmm, so idl0r seems MIA? :)

**Comment by mgorny** [comment]
Well, he sez he didn't use VDR for a while. The whole thing seems simple enough, so I'll just merge it without more delay.

File: media-plugins/vdr-ddci2/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>d.scheller.oss@gmail.com</email>
+		<name>Daniel Scheller</name>
+	</maintainer>
+	<upstream>
```
Comment (mgorny) [inline media-plugins/vdr-ddci2/metadata.xml]: For the future: please remember to add proxy-maint, so we know to commit for you ;-). I'll fix it now.

Comment (herrnst) [inline media-plugins/vdr-ddci2/metadata.xml]: Thanks for pointing this out, will do next time (as mentioned, first-time contribution :) ) - and thanks for merging!

**Pull request closed without merging**